module1/assessments/index.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: page
+title: Module 1 Assessments
+---
+
+* [Week 1](/module1/assessments/week1-3168.md)
```

Comment:
Are these visible to the students? Or hidden with a `showInstructorNotes` ?

Reply:
This page is an orphan page; it's live, but there's no direct link from any other page on the website. I created it mainly for us, but I can stash them somewhere else if you think I should remove it.